src/sentry/api/endpoints/organization_projects.py

```diff
@@ -78,9 +78,8 @@ def get(self, request, organization):
                     }, status=400
                 )
         else:
-            team_list = list(request.access.teams)
             queryset = Project.objects.filter(
-                teams__in=team_list,
```

Comment:
id be a bit concerned that this somewhat causes a regression -- the other paths all restrict based on access

whats the end need for it?